File: Aulas/Aula_46_2307/aula46-2/src/main/java/com/programando/aula462/model/Jogo.java
package com.programando.aula462.model;

public class Jogo {
    private int id;
    private String nome;
    private String descricao;
    private double preco;
    private String imagem;
    private String site;

    public Jogo() {}

    public Jogo(int id, String nome, String descricao, double preco, String imagem, String site) {
        this.id = id;
        this.nome = nome;
        this.descricao = descricao;
        this.preco = preco;
        this.imagem = imagem;
        this.site = site;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public double getPreco() {
        return preco;
    }

    public void setPreco(double preco) {
        this.preco = preco;
    }

    public String getImagem() {
        return imagem;
    }

    public void setImagem(String imagem) {
        this.imagem = imagem;
    }

    public String getSite() {
        return site;
    }

    public void setSite(String site) {
        this.site = site;
    }
}

File: Aulas/Aula_46_2307/aula46-1/src/app/models/jogo.ts
export class Jogo{
  public id!: number;
  public nome!: string;
  public descricao!: string;
  public preco!: number;
  public imagem!: string;
  public site!: string;

  constructor(id: number, nome: string, descricao: string, preco: number, imagem: string, site: string){
    this.id = id;
    this.nome = nome;
    this.descricao = descricao;
    this.preco = preco;
    this.imagem = imagem;
    this.site = site;
  }
}
